tip-router-operator-cli/src/lib.rs

```diff
@@ -113,53 +132,128 @@ fn emit_inconsistent_tree_node_amount_dp(
     if actual_claims > expected_claims {
         datapoint_error!(
             "tip-distributor",
-            (
-                "actual_claims_exceeded",
-                format!("tip_distribution_account={tip_distribution_account},actual_claims={actual_claims}, expected_claims={expected_claims}"),
-                String
-            ),
+            ("actual_claims_exceeded", format!(
+                "tip_distribution_account={tip_distribution_account},actual_claims={actual_claims}, expected_claims={expected_claims}"
+            ), String)
         );
     } else {
-        datapoint_warn!(
-            "tip-distributor",
-            (
-                "actual_claims_below",
-                format!("tip_distribution_account={tip_distribution_account},actual_claims={actual_claims}, expected_claims={expected_claims}"),
-                String
+        datapoint_warn!("tip-distributor", (
+            "actual_claims_below",
+            format!(
+                "tip_distribution_account={tip_distribution_account},actual_claims={actual_claims}, expected_claims={expected_claims}"
             ),
-        );
+            String
+        ));
     }
 }
 
 impl GeneratedMerkleTreeCollection {
-    pub fn new_from_stake_meta_collection(
+    pub async fn new_from_stake_meta_collection(
```

Comment:
why is this re-implemented? Could we not import the `GeneratedMerkleTreeCollection` from `generated_merkle_tree` and utilize the existing implementation rather than copy/pastaing?

Reply:
Will revisit this tomorrow